### ISSUE TITLE
Congo Bad Geometry

### ISSUE THREAD

**Issue by justinelliotmeyers** [facebookmicrosites/Open-Mapping-At-Facebook#27]
**Title:** random straight Jogs north side of Congo (Brazzaville)

#
**Files:** 

[List the files related to this issue](https://mapwith.ai/country_exports/CG_mapwithai_road_data.gpkg.tar.gz)

#
**Description:**

![image](https://github.com/facebookmicrosites/Open-Mapping-At-Facebook/assets/5324316/98a671ea-200d-4bb4-9d40-987b39648b75)

**Comment by tsmock** [comment]
These aren't random straight jogs.

It appears that those lines (largely) follow tile boundaries at z12.

**Comment by justinelliotmeyers** [comment]
Thanks for the quick response. I figured it was something like that. I looked at recent sentinel data and nothing follows them. Thanks for confirming the tile thing. 

Can they be removed or are they not random then?

**Comment by tsmock** [comment]
I don't believe we have a process for removing them. I don't believe they are random; I rather suspect that the tiles have some kind of boundary that looks like a road. It might be those tiles had clouds, while the surrounding tiles did not have clouds.

Rapid has a ticket open for saving the `Remove this feature` response (see https://github.com/facebook/Rapid/issues/98 ). That might be what will allow the removal of those false positives in the future.
